ansible_collections/arista/avd/roles/eos_designs/templates/base/base.py

```diff
@@ -0,0 +1,559 @@
+                daemon_terminattr.update({'cvauth': {'method': 'token-secure',
+                                                     'token_file': get(self._hostvars, 'cvp_token_file',
+                                                                       '/tmp/cv-onboarding-token')}})
+
+            daemon_terminattr.update({'cvvrf': self._mgmt_interface_vrf})
+            daemon_terminattr.update({'smashexcludes': get(self._hostvars, 'terminattr_smashexcludes')})
+            daemon_terminattr.update({'ingestexclude': get(self._hostvars, 'terminattr_ingestexclude')})
+            daemon_terminattr.update({'disable_aaa': get(self._hostvars, 'terminattr_disable_aaa', False)})
+            return daemon_terminattr
+        return None
+
+    @cached_property
+    def vlan_internal_order(self):
+        range = get(self._internal_vlan_order, 'range')
+        return {
+            'allocation': get(self._internal_vlan_order, 'allocation'),
+            'range': {
+                'beginning': get(range, 'beginning'),
+                'ending': get(range, 'ending')
+            }
+        }
+
+    @cached_property
+    def event_monitor(self):
+        if get(self._hostvars, 'event_monitor') is True:
+            return {'enabled': 'true'}
+        return None
+
+    @cached_property
+    def event_handlers(self):
+        event_handlers = get(self._hostvars, 'event_handlers')
+        if event_handlers is not None and hasattr(event_handlers, '__iter__'):
+            event_handlers = self._convert_dicts(event_handlers, 'name')
+            for handler in event_handlers:
+                handler_name = get(handler, 'name')
+                event_handler = {handler_name: {}}
+                action = get(handler, 'action')
+                action_type = get(handler, 'action_type')
+                delay = get(handler, 'delay')
+                trigger = get(handler, 'trigger')
+                regex = get(handler, 'regex')
+                if action is not None and action_type is not None:
+                    event_handler[handler_name].update({'action_type': action_type})
+                    event_handler[handler_name].update({'action': action})
+                if delay is not None:
+                    event_handler[handler_name].update({'delay': delay})
+                if get(handler, 'asynchronous') is True:
+                    event_handler[handler_name].update({'asynchronous': 'handler.asynchronous'})
+                if trigger is not None:
+                    event_handler[handler_name].update({'trigger': trigger})
+                if regex is not None:
+                    event_handler[handler_name].update({'regex': regex})
+            return event_handler
+        return None
+
+    @cached_property
+    def load_interval(self):
+        load_interval_default = get(self._hostvars, 'load_interval_default')
+        if load_interval_default is not None:
+            return {'default': load_interval_default}
+        return None
```

Comment:
Starting python 3.8 we could write this
```suggestion
        if (load_interval_default := get(self._hostvars, 'load_interval_default')) is not None:
            return {'default': load_interval_default}
        return None
```

Reply:
If we choose to do this here, we should do it all over.

Reply:
hmm so maybe not worse doing it then